go/client/logclient.go

```diff
@@ -227,20 +242,18 @@ func (c *LogClient) addChainWithRetry(ctx context.Context, ctype ct.LogEntryType
 		req.Chain = append(req.Chain, link)
 	}
 	httpStatus := "Unknown"
-	// Retry after 1s, 2s, 4s, 8s, 16s, 32s, 64s, 128s, 128s, ....
+	// Retry after 1s, 2s, 4s, 8s, 16s, 32s, 64s, 128s, 128s
```

Comment:
Please can you remove the back-out related changes from this PR, and keep it purely about logging?